dataclasses_avroschema/model_generator/generator.py

```diff
@@ -447,7 +436,7 @@ def get_field_default(self, *, field_type: str, default: typing.Any, name: str)
         elif field_type == field_utils.BYTES:
             return f'b"{default}"'
         elif isinstance(field_type, dict) and field_type.get("type") == field_utils.ENUM:
-            return f"{stringcase.pascalcase(name)}.{stringcase.uppercase(default)}"
+            return f"{stringcase.pascalcase(field_type['name'])}.{stringcase.uppercase(default)}"
```

Comment:
Instead of a correct `my_color = Color.RED`, this would previously generate `my_color = MyColor.RED`.